app/src/main/res/values/settings_defaults_dont_translate.xml

```diff
@@ -28,7 +28,7 @@
     <bool name="settings_default_hide_soft_when_physical">true</bool>
     <bool name="settings_default_enable_alt_space_language_shortcut">true</bool>
     <bool name="settings_default_enable_shift_space_language_shortcut">true</bool>
-    <bool name="settings_default_key_press_shows_preview_popup">true</bool>
+    <bool name="settings_default_key_press_shows_preview_popup">false</bool>
```

Comment:
I rather not. I think that most people expect to see a pop-up while typing.

Reply:
This is apart of what I said in my issue I opened, the current pop up behaviour is disturbing it is not natural. I checked and people liked it, The new behaviour feels good and natural. It has pop ups, the popup is a visual effect, the one above, not tooltip.